lib/resources/r4/slot_patch.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Related Person
+field_name_base_url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson
+operations:
+  - name: status
+    path: /status
+    operation: replace
+    type: code
+    url: https://hl7.org/fhir/R4/slot-definitions.html#Slot.status
+    description: Replace the slot status.
+    example: |
+      {
+        "status": "free"
+      }
+    note: Currently supports only `free` and `busy tentative`.
+    
```

Comment:
lets add a new line.

Reply:
im confused where do we need a new line?

Reply:
Initially, we were missing newline at the end of file as shown here: https://github.com/cerner/fhir.cerner.com/pull/608/commits/55ecd03d3c4d20aa06ab1960a2fb7aa33744bdaf#diff-8a92cfc782cbec26cd0d479d0184b0a249e166900d9f3bbef43ea04612760a9aR15

Looks like it was added in one of the subsequent commits, so resolving it.